### PR TITLE
Build with Go 1.14

### PR DESCRIPTION
**Pull request by lufia**
I upgraded Go toolchain, 1.12 -> 1.14, and rename **appveyor.yml**.
See https://github.com/mackerelio/go-osstat/pull/12#issuecomment-522956868

My previous p-r #228 is closed because there is some issues worried for net/http in go1.13. these is fixed in go1.14.1.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+  VERSION: go1.14.1
   GOPATH: c:\gopath
-  GOROOT: c:\go112
+  GOROOT: c:\go
   GO111MODULE: on
 build: off
 install:
+  - rmdir %GOROOT% /s /q
+  - appveyor DownloadFile https://dl.google.com/go/%VERSION%.windows-amd64.zip
+  - 7z x %VERSION%.windows-amd64.zip -oC:\
```
Comment (lufia) [inline .appveyor.yml:12]: The official archive's top directory is **go/**, and *7z* cannot skip top directory. Thus we should remove **C:\go** then extract the archive into **C:\go**.

File: .appveyor.yml [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 environment:
+  VERSION: go1.14.1
   GOPATH: c:\gopath
-  GOROOT: c:\go112
+  GOROOT: c:\go
   GO111MODULE: on
 build: off
 install:
+  - rmdir %GOROOT% /s /q
+  - appveyor DownloadFile https://dl.google.com/go/%VERSION%.windows-amd64.zip
```
Comment (astj) [inline .appveyor.yml:11]: Can/should we cache this file? (Or does it fast enough?)

File: .appveyor.yml (hunk 1 above)
Comment (astj) [reply .appveyor.yml:12]: It's acceptable, but I think `GOROOT` can be set to somewhere else, like `/opt/xxx` in *nix.

File: .appveyor.yml (hunk 2 above)
Comment (lufia) [reply .appveyor.yml:10]: Though AppVeyor has cache directive, I'm not sure how to skip downloading the archive on windows batch script! sorry.

File: .appveyor.yml (hunk 1 above)
Comment (lufia) [reply .appveyor.yml]: There is C:\tools directory by default. GOROOT was moved to it.

**Pull request merged**